### PR TITLE
SEO-174775 WPF Page with Redirect-

### PR DESCRIPTION
**Pull request by Yvone-Atieno**
Hi @Aishwarya-Ganesan,

Have changed WPF Page with Redirect, please review.

Regards,
Yvone.

### CODE REVIEW

File: wpf/DataGrid/Grouping.md [hunk 1]
```diff
@@ -566,10 +566,10 @@ The [GroupChangingEventArgs](https://help.syncfusion.com/cr/wpf/Syncfusion.UI.Xa
 
 [Group](https://help.syncfusion.com/cr/wpf/Syncfusion.UI.Xaml.Grid.GroupChangingEventArgs.html#Syncfusion_UI_Xaml_Grid_GroupChangingEventArgs_Group) - Gets the group that’s being expanded.
 
-[Cancel](https://docs.microsoft.com/en-us/dotnet/api/system.componentmodel.canceleventargs.cancel?view=net-5.0) – Decides whether to cancel the group expansion.
+[Cancel](https://learn.microsoft.com/en-us/dotnet/api/system.componentmodel.canceleventargs.cancel?view=net-5.0) – Decides whether to cancel the group expansion.
  
-You can cancel the group expansion by setting [GroupChangingEventArgs.Cancel](https://docs.microsoft.com/en-us/dotnet/api/system.componentmodel.canceleventargs.cancel?view=net-5.0) to `true`.
-
+You can cancel the group expansion by setting [GroupChangingEventArgs.Cancel]() to `true`.
+https://learn.microsoft.com/en-us/dotnet/api/system.componentmodel.canceleventargs.cancel?view=net-5.0
```
Comment (MallikaRK) [inline wpf/DataGrid/Grouping.md:572]: Seems you mistakenly placed the URL. Update the URL in needed place.

**Review by MallikaRK — CHANGES_REQUESTED** [review]
Commented

File: wpf/DataGrid/Grouping.md (hunk 1 above)
Comment (Yvone-Atieno) [reply wpf/DataGrid/Grouping.md]: updated

**Review by MallikaRK — APPROVED** [review]
Approved

**Review by Aishwarya-Ganesan — APPROVED** [review]
Changes are fine.

**Pull request merged**